File: vs2003_cd01/Program Files/Microsoft Visual Studio .NET 2003/VC#/VC#Wizards/CSharpWebControlWiz/Templates/1033/webcontrol.cs
using System;
using System.Web.UI;
using System.Web.UI.WebControls;
using System.ComponentModel;

namespace [!output SAFE_NAMESPACE_NAME]
{
	/// <summary>
	/// Summary description for [!output SAFE_CLASS_NAME].
	/// </summary>
	[DefaultProperty("Text"),
		ToolboxData("<{0}:[!output SAFE_CLASS_NAME] runat=server></{0}:[!output SAFE_CLASS_NAME]>")]
	public class [!output SAFE_CLASS_NAME] : System.Web.UI.WebControls.WebControl
	{
		private string text;

		[Bindable(true),
			Category("Appearance"),
			DefaultValue("")]
		public string Text
		{
			get
			{
				return text;
			}

			set
			{
				text = value;
			}
		}

		/// <summary>
		/// Render this control to the output parameter specified.
		/// </summary>
		/// <param name="output"> The HTML writer to write out to </param>
		protected override void Render(HtmlTextWriter output)
		{
			output.Write(Text);
		}
	}
}

File: vs2003_cd01/Program Files/Microsoft Visual Studio .NET 2003/Vb7/VBWizards/WebControl/Templates/1033/WebControl.vb
Imports System.ComponentModel
Imports System.Web.UI

<DefaultProperty("Text"), ToolboxData("<{0}:[!output SAFE_ITEM_NAME] runat=server></{0}:[!output SAFE_ITEM_NAME]>")> Public Class [!output SAFE_ITEM_NAME]
    Inherits System.Web.UI.WebControls.WebControl

    Dim _text As String

    <Bindable(True), Category("Appearance"), DefaultValue("")> Property [Text]() As String
        Get
            Return _text
        End Get

        Set(ByVal Value As String)
            _text = Value
        End Set
    End Property

    Protected Overrides Sub Render(ByVal output As System.Web.UI.HtmlTextWriter)
        output.Write([Text])
    End Sub

End Class
